setup.cfg

```diff
@@ -55,6 +55,7 @@ dev =
 
 pyscf =
     pyscf~=2.2.0
+    scipy<1.11.0
```

Comment:
@max-radin is this a fix for what's happening here: https://github.com/zapatacomputing/benchq/pull/87#issuecomment-1611773425 ?

Reply:
It seems so!

Reply:
Yes I believe so. I think we'll have to constrain the scipy version until pyscf adds support for 1.11.